cucumber-rails.gemspec

```diff
@@ -47,7 +46,6 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.4.0'
   s.rubygems_version = '>= 1.6.1'
-  s.files            = Rake::FileList['**/*'].exclude(*File.read('.gitignore').split)
-  s.test_files       = Dir['{spec,features}/**/*']
```

Comment:
Do we not want refs to these?

Reply:
I don't think so, maybe? Do you want to have `test_files` defined? I personally do so, but it's okay to not do that, really. So your call 😄

Reply:
FYI: Any files specified in `test_files` will be added to the gem package.

Is there an expectation for downstream `cucumber-rails` users to run these tests?

Reply:
Most of the downstream maintainers run upstream tests to see if there's no regression or so. So we definitely need tests. That said, we, in Debian, pull from the GitHub source instead of rubygems for this case, so it's okay to not add here. Or if you want to add for consistency purposes, that's okay (in fact, good) as well. So I'm okay with either, really 😅